showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+
+In Estonia the SARS-CoV-2 sequencing was carried out in collaboration with the SYNLAB Eesti, Health Board Estonia and University of Tartu. The Synlab provided information on the basic sample metadata and PCR outcome, Health Board provided sample metadata on e.g. age, gender, county, vaccination information. 
+
+The central data storage was at GitLab instance at the University of Tartu. The University of Tartu research groups merged the data, created plate information files for sequencing, collecting the sequencing results, generated custom reports, interactive dashboards and brokered data to ENA. In addition to data sequenced in Estonia, also data from ECDC was analysed in similar manner to provide accurate overview of the situation.
+
+### The aim of KoroGeno-EST
+The aim of KoroGeno-EST-3 and KoroGeno-EST-2022 (hereafter, Study) studies was to sequence and conduct molecular-epidemiology analysis for more than 15 000 SARS-CoV-2 full genomes from 1 May to 31 December 2022 in Estonia. The number of weekly sequenced strains depended directly on the epidemiological situation, including the number of newly diagnosed SARS-CoV-2 cases, as well as the need to monitor clinically, demographically, or mutationally significant strains. 
+
+Based on the sequenced samples, weekly reports were prepared for the [Republic of Estonia Health Board](https://www.terviseamet.ee/en) (hereafter, Health Board (HB)) and "[European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en)". In collaboration with the HB, SARS-CoV-2 genomic data was used to analyze infection clusters to trace newly infected subjects. 
+
+In addition, the study identified the prevalence of variants associated with higher pathogenicity or infectiousness or corresponding mutations. Based on these data, authorities got a better overview of circulating SARS-CoV-2 strains as well as imported cases. In general, the current study made a significant contribution to the governmental decision-making process and helped more efficiently suppress the spread of the COVID-19 epidemic.
+
+## Metadata and sample acquisition
+
+Throughout most of the pandemic, SYNLAB Eesti testing sites in Estonia provided free SARS-CoV-2 PCR testing for all individuals living in the country. The free testing stopped for general population in May 2022 and for risk groups in November 2022. Self-paid testing has been available since 2020.
```

Comment:
```suggestion
Throughout most of the pandemic, SYNLAB Eesti testing sites in Estonia provided free SARS-CoV-2 PCR testing for all individuals living in the country. Free testing was discontinued for general population in May 2022 and for risk groups in November 2022. Self-paid testing has been available since 2020.
```

Reply:
ok